capabilities/generative-organizational-culture.md

```diff
@@ -1,62 +1,89 @@
+
+Don't worry if the description doesn't exactly match your situation. These descriptions are meant to be examples of situations that would qualify for the associated score.
+
+1. Top-Down: Teams operate under a highly directive approach, with leadership providing explicit instructions and priorities. Autonomy for decision-making is limited. In the event of failure, the focus is on individual accountability and administering corrective action.
+2. Bureaucratic: Teams follow established procedures and protocols, with clear roles and responsibilities. Sometimes the specific instructions are vague or incomplete with no clear product leader. Teams have some flexibility to adapt to changing circumstances. However, leadership approval is still required for most meaningful decisions.
+3. Collaborative: Teams seek input and expertise from other teams to inform their decisions, but maintain clear ownership and responsibility for their work. Each team has some autonomy to make decisions within established boundaries. However, strategic direction is set by leadership, and teams are expected to align their work with these top-down priorities.
+4. Generative: Teams seek input and expertise from other teams to inform their decisions, but maintain clear ownership and responsibility for their work. Each team has some autonomy to make decisions within established boundaries. Strategic direction is set by leadership, but factors in ground-level feedback from the teams.
+
+The number you selected represents your overall score for this capability. If you feel like your organization fits somewhere in between two scores, it's okay to use a decimal. For example, if you think your organization's culture is somewhere between bureaucratic and collaborative, you would score a 2.5.
+
+Generally, an overall score equal to or less than 3 means you'll likely gain a lot of value from experimenting with some of the supporting practices listed here. An overall score higher than 3 generally means your organization is largely proficient, or well on its way to becoming proficient, in the area of Generative Organizational Culture; you would likely benefit from evaluating your scores in other capabilities.
+
 
 ## Supporting Practices
+The following is a curated list of supporting practices to consider when looking to improve your team's Generative Organizational Culture capability. While not every practice will be beneficial in every situation, this list is meant to provide teams with fresh, pragmatic, and actionable ideas to support this capability.
 
-### Conduct Blameless Retrospectives
+### Conduct Blameless Retrospectives [link to "conduct retrospective meetings" practice?]
```

Comment:
Supporting Practices are not linked here, as they are in other Capabilities. Are these existing practices, just referred to by another name? If a Supporting Practice struck me as familiar, I added a note in brackets about linking it to the relevant practice.

Reply:
leave without link till done.